tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -45,7 +47,7 @@
 titles (or sometimes even ticklabels) go outside the figure area, and are thus
 clipped.
 
-"""
+""".format(version=matplotlib.__version__)
```

Comment:
How about simply removing this annotation? This will need to be updated once constrained layout is not experimental anyways.

Reply:
Not quite sure what you meant; I've just got rid of the version number now, is that okay?